src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const DISCOUNT_50 = 50;
```

Comment:
No need to add what is stored there in the variable names, because the number of days can change and you will have to fix it in the entire code, not just in the variable definition.